### PR TITLE
NOJIRA Arrange AngularJS pieces by UX view rather than Angular type

### PR DESCRIPTION
**Pull request by raydavis**
SuiteC-style as opposed to CalCentral-style.

### CODE REVIEW

File: boac/templates/index.html [hunk 1]
```diff
@@ -18,14 +18,14 @@
...
-    <script src="/static/js/app.js"></script>
-    <script src="/static/js/routes.js"></script>
-    <script src="/static/js/controllers/authController.js"></script>
-    <script src="/static/js/controllers/cohortController.js"></script>
-    <script src="/static/js/controllers/landingController.js"></script>
-    <script src="/static/js/controllers/studentController.js"></script>
-    <script src="/static/js/factories/analyticsFactory.js"></script>
-    <script src="/static/js/factories/authFactory.js"></script>
-    <script src="/static/js/factories/cohortFactory.js"></script>
+    <script src="/static/app/app.js"></script>
```
Comment (johncrossman) [inline boac/templates/index.html]: @raydavis This looks good to me. One thought: rename `app.js` to `bootstrap.js`?  (I'll let @paulkerschen review, too.)

Comment (raydavis) [reply boac/templates/index.html]: Thank you, @johncrossman ; I like that thought.

File: tox.ini [hunk 2]
```diff
@@ -18,7 +18,7 @@ commands =
 [testenv:lint-js]
 commands =
     {toxinidir}/scripts/install-tox-deps.sh
-    eslint --color {posargs:boac/static/js}
+    eslint --color {posargs:'boac/static/app/**/*.js'}
```
Comment (raydavis) [inline tox.ini]: Since I'm still detoxed, this is a guess at the proper syntax (although both "eslint" and "stylelint" work for me from the command line).

**Comment by johncrossman** [comment]
Alley-oop!

**Pull request merged**